app/models/employee/employee.server.tsx

```diff
@@ -0,0 +1,104 @@
+import { Employee, Role, User } from "@prisma/client";
+import { database } from "~/helpers/db-helper.server";
+
+export const findEmployees = async () => {
+  return database.employee.findMany({
+    include: {
+      user: true,
+    },
+  });
+};
+
+export const findEmployeeById = async (
+  id: string
+): Promise<Employee | null> => {
+  return database.employee.findUnique({
+    where: {
+      id,
+    },
+    include: {
+      user: true,
+    },
+  });
+};
+
+type EmployeeCreateInput = {
+  email: string;
+  password: string;
+  firstName: string;
+  lastName: string;
+  role: Role;
+};
+
+export const createEmployee = async (
+  email: string,
+  password: string,
+  firstName: string,
+  lastName: string,
+  role: Role
+) => {
```

Comment:
Use the type `EmployeeCreateInput` as a parameter in the function `createEmployee` instead of the single string parameters or remove the unused type